testfiles/HeatPumpVSAS_variableCrankcaseHeaterCapacity.idf

```diff
@@ -0,0 +1,1798 @@
+    HPACHeatEIRFFF,          !- Speed 10 Energy Input Ratio Function of Air Flow Fraction Curve Name
+    heaterCapCurve;
+
+  Curve:Biquadratic,
+    HPACHeatCapFT,           !- Name
+    0.8529681407,            !- Coefficient1 Constant
+    -0.0004847169,           !- Coefficient2 x
+    -0.0000010693,           !- Coefficient3 x**2
+    0.0185542164,            !- Coefficient4 y
+    0.0000872425,            !- Coefficient5 y**2
+    -0.0000166868,           !- Coefficient6 x*y
+    17.78,                   !- Minimum Value of x
+    23.33,                   !- Maximum Value of x
+    -28.89,                  !- Minimum Value of y
+    17.22,                   !- Maximum Value of y
+    0.3799,                  !- Minimum Curve Output
+    1.1896,                  !- Maximum Curve Output
+    Temperature,             !- Input Unit Type for X
+    Temperature,             !- Input Unit Type for Y
+    Dimensionless;           !- Output Unit Type
+
+  Curve:Biquadratic,
+    HPACHeatEIRFT,           !- Name
+    0.7077081462,            !- Coefficient1 Constant
+    0.0148163478,            !- Coefficient2 x
+    0.0002622589,            !- Coefficient3 x**2
+    -0.0113239622,           !- Coefficient4 y
+    0.0002939277,            !- Coefficient5 y**2
+    -0.0003605284,           !- Coefficient6 x*y
+    17.78,                   !- Minimum Value of x
+    23.33,                   !- Maximum Value of x
+    -28.89,                  !- Minimum Value of y
+    17.22,                   !- Maximum Value of y
+    0.8266,                  !- Minimum Curve Output
+    2.0277,                  !- Maximum Curve Output
+    Temperature,             !- Input Unit Type for X
+    Temperature,             !- Input Unit Type for Y
+    Dimensionless;           !- Output Unit Type
+
+!-   ===========  ALL OBJECTS IN CLASS: COIL:COOLING:DX:VARIABLESPEED ===========
+
+  Coil:Cooling:DX:VariableSpeed,
+    Heat Pump ACDXCoil 1,    !- Name
+    DX Cooling Coil Air Inlet Node,  !- Indoor Air Inlet Node Name
+    Heating Coil Air Inlet Node,  !- Indoor Air Outlet Node Name
+    10.0,                    !- Number of Speeds {dimensionless}
+    10.0,                    !- Nominal Speed Level {dimensionless}
+    32000,                   !- Gross Rated Total Cooling Capacity At Selected Nominal Speed Level {W}
+    1.7,                     !- Rated Air Flow Rate At Selected Nominal Speed Level {m3/s}
+    0.0,                     !- Nominal Time for Condensate to Begin Leaving the Coil {s}
+    0.0,                     !- Initial Moisture Evaporation Rate Divided by Steady-State AC Latent Capacity {dimensionless}
+    HPACCOOLPLFFPLR,         !- Energy Part Load Fraction Curve Name
+    ,                        !- Condenser Air Inlet Node Name
+    AirCooled,               !- Condenser Type
+    ,                        !- Evaporative Condenser Pump Rated Power Consumption {W}
+    ,                        !- Crankcase Heater Capacity {W}
```

Comment:
For Coil:Cooling:DX:VariableSpeed, the crankcase heater as a function of temperature curve needs to be moved here. And a value for Crankcase Heater Capacity should be entered, otherwise CC heater power will be 0. A transition rule will be needed.